spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -296,6 +297,10 @@ public AutoTimeProperties getAutotime() {
 		 */
 		private final Map<String, String> maximumExpectedValue = new LinkedHashMap<>();
 
+		private final Map<String, Duration> expiry = new LinkedHashMap<>();
```

Comment:
This should have a description like the other fields.

Reply:
Please review the description.